.github/workflows/respec.yaml

```diff
@@ -0,0 +1,61 @@
+name: respec
+
+# author: @MikeRalphson
+# issue: https://github.com/OAI/OpenAPI-Specification/issues/1564
+
+#
+# This workflow updates the respec 'pretty' rendered versions of the spec 
+# on the gh-pages branch when the corresponding markdown files change.
+#
+
+# run this on push to main
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch: {}
+
+jobs:
+  respec:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4 # checkout main branch
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-node@v4 # setup Node.js
+      with:
+        node-version: '20.x'
+    
+    - name: Install dependencies
+      run: npm ci
+
+    - uses: actions/checkout@v4 # checkout gh-pages branch
+      with:
+        token: ${{ secrets.OAS_REPO_TOKEN }}
+        repository: ralfhandl/OpenAPI-Specification  # TODO: change to OAI/...
```

Comment:
Based on your comment, I assume that means this TODO must be fixed after merge. I'll take your word on that ...

Reply:
Correct, otherwise we'd open PRs in https://github.com/OAI/OpenAPI-Specification/pulls for publishing the Overlay specification on https://spec.openapis.org/, which we probably don't want before we release v1.0.